app/src/main/java/fr/free/nrw/commons/campaigns/CampaignsPresenter.java

```diff
@@ -108,7 +108,7 @@ public void getCampaigns() {
                     }
 
                     @Override public void onError(Throwable e) {
-                        Log.e(TAG, "could not fetch campaigns: " + e.getMessage());
+                        Timber.tag(TAG).e("could not fetch campaigns: %s", e.getMessage());
```

Comment:
Is TAG really necessary here? I thought Timber automatically adds the class name in place of a tag. Again, this might not be something you introduced, but it would be good if we can fix it altogether.

Reply:
I will look in to this further and update the code.